lib/create-voice-server.js

```diff
@@ -45,3 +45,5 @@ module.exports = function createVoiceServer(handle) {
     server.on('connection', onConnection.bind(null, server));
   });
 };
+
+/** @typedef {import("events").EventEmitter} EventEmitter */
```

Comment:
`@types/node` doesn't add EventEmitter globally. Alternatively to this we could `const {EventEmitter} = require('events')` at the top of the file.

Reply:
But that alternative leaves a dangling import. Since EventEmitter isn't used in the code, only in comments.

Reply:
Thanks for offering alternatives, but I agree with the choice you've made here. I prefer declaring types at the top of the file, though.

Reply:
I prefer declaring types at the top as well. Applying that how should we manage what the comment modifies? Like in this file if it is put after the imports, it would be documenting `onConnection`.

I see two solutions for that while keeping the type in the file:
1. Add a comment documenting a line of code between typedef and none code comments.
2. Put comments like these at the end of the file.

To make the PR smaller and easier to review I chose the second.

I'll put these typedefs to the top and add comments like stated in 1. Does that sound good? Do you have something else in mind?

Reply:
Hrm. Are you asking for a change to be made here? On second review of your comment I'm not sure if you are commenting on it or asking for a change.

If so, I'dj be happy to do the stroke-out section I stated:
> I'll put these typedefs to the top and add comments like stated in 1. Does that sound good? Do you have something else in mind?

Reply:
I was asking for a change, although I thought `@typedef` always works as a standalone declaration. If we have to insert some weird comment as a delimiter, then I guess that's still slightly better than having types squirreled away at the bottom. Is a delimiter like that really necessary, though? On my machine, the compiler seems fine with this change:

```diff
diff --git a/lib/create-voice-server.js b/lib/create-voice-server.js
index c1386c4..e14f78d 100644
--- a/lib/create-voice-server.js
+++ b/lib/create-voice-server.js
@@ -2,6 +2,8 @@
 
 const net = require('net');
 
+/** @typedef {import("events").EventEmitter} EventEmitter */
+
 const onConnection = (server, socket) => {
   let emitted = '';
   socket.on('data', buffer => (emitted += buffer.toString()));
@@ -46,4 +48,3 @@ module.exports = function createVoiceServer(handle) {
   });
 };
 
-/** @typedef {import("events").EventEmitter} EventEmitter */
```

Reply:
I could also be wrong and remembering something from an older version of typescript? I retract that stuff I said.

(It's pretty nice to see I'm wrong on this. I super thought this was a thing.)